schemas/alpha/azure.yaml.json

```diff
@@ -678,6 +678,11 @@
                     "items": {
                         "type": "string"
                     }
+                },
+                "remoteBuild": {
+                    "type": "boolean",
+                    "title": "Optional. Whether to build the image remotely",
+                    "description": "If set to true, the image will be built remotely using the Azure Container Registry remote build feature. If set to false, the image will be built locally using Docker."
                 }
```

Comment:
Should we consider adding that this can only be used when using an ACR or do you feel it is implied?

Reply:
I feel like it is implied today by saying "using the Azure Container Registry remote build feature", but if we end up getting a lot of feedback that remote build doesn't work when using non ACR registries (and I expected it to) I will look at updating the prose here.